test/autoschedulers/mullapudi2016/reorder.cpp

```diff
@@ -184,6 +211,11 @@ int main(int argc, char **argv) {
     }
 
     {
+        if (get_jit_target_from_environment().has_gpu_feature()) {
+            std::cout << "Mullapudi for GPU test for Test Case 3 skipped because of reordering bug.\n";
+            return 0;
+        }
+
```

Comment:
Move this just out of the `{ }` block, remove the `return`, and use ` else ` to then enter the actual test. Code flow wise, it feels much cleaner; disabling one test shouldn't depend on the fact that it's the last one.